modules/chre/CMakeLists.txt

```
@@ -0,0 +1,104 @@
+# Copyright (c) 2021 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_CHRE)
+  set(CHRE_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  zephyr_library()
+
+  set(CHRE_INCLUDE_DIRS
+      "${CHRE_DIR}/chre_api/include"
+      "${CHRE_DIR}/chre_api/include/chre_api"
+      "${CHRE_DIR}/core/include"
+      "${CHRE_DIR}/platform/include"
+      "${CHRE_DIR}/util/include"
+      "${CMAKE_CURRENT_SOURCE_DIR}/include")
+  zephyr_include_directories("${CHRE_INCLUDE_DIRS}")
+  zephyr_library_include_directories("${CHRE_INCLUDE_DIRS}")
+
+  zephyr_library_sources(
+      "src/log_module.c"
+      "${CHRE_DIR}/core/debug_dump_manager.cc"
+      "${CHRE_DIR}/core/event.cc"
+      "${CHRE_DIR}/core/event_loop.cc"
+      "${CHRE_DIR}/core/event_loop_manager.cc"
+      "${CHRE_DIR}/core/event_ref_queue.cc"
+      "${CHRE_DIR}/core/host_comms_manager.cc"
+      "${CHRE_DIR}/core/init.cc"
+      "${CHRE_DIR}/core/nanoapp.cc"
+      "${CHRE_DIR}/core/settings.cc"
+      "${CHRE_DIR}/core/static_nanoapps.cc"
+      "${CHRE_DIR}/core/timer_pool.cc"
+      "${CHRE_DIR}/platform/shared/version.cc"
+  )
+
+  # Optional audio support
+  if(DEFINED CHRE_AUDIO_SUPPORT_ENABLED)
```

I think there are a few places where you need to prefix the Kconfig symbols with "CONFIG_"
```suggestion
if(CONFIG_CHRE_AUDIO_SUPPORT_ENABLED)
```

---

Thanks, fixed

---

Done